data-visualize.qmd

```diff
@@ -203,27 +207,19 @@ Before we add more layers to this plot, let's pause for a moment and review the
 
 > Removed 2 rows containing missing values (`geom_point()`).
 
-We're seeing this message because there are two penguins in our dataset with missing body mass and flipper length values and ggplot2 has no way of representing them on the plot.
-You don't need to worry about understanding the following code yet (you will learn about it in @sec-data-transform), but it's one way of identifying the observations with `NA`s for either body mass or flipper length.
-
-```{r}
-penguins |>
-  select(species, flipper_length_mm, body_mass_g) |>
-  filter(is.na(body_mass_g) | is.na(flipper_length_mm))
-```
-
+We're seeing this message because there are two penguins in our dataset with missing body mass and/or flipper length values and ggplot2 has no way of representing them on the plot without both of these values.
 Like R, ggplot2 subscribes to the philosophy that missing values should never silently go missing.
 This type of warning is probably one of the most common types of warnings you will see when working with real data -- missing values are a very common issue and you'll learn more about them throughout the book, particularly in @sec-missing-values.
 For the remaining plots in this chapter we will suppress this warning so it's not printed alongside every single plot we make.
 
 ### Adding aesthetics and layers
 
-Scatterplots are useful for displaying the relationship between two variables, but it's always a good idea to be skeptical of any apparent relationship between two variables and ask if there may be other variables that explain or change the nature of this apparent relationship.
+Scatterplots are useful for displaying the relationship between two numerical variables, but it's always a good idea to be skeptical of any apparent relationship between two variables and ask if there may be other variables that explain or change the nature of this apparent relationship.
```

Comment:
```suggestion
Scatterplots are useful for displaying the relationship between two numeric variables, but it's always a good idea to be skeptical of any apparent relationship between two variables and ask if there may be other variables that explain or change the nature of this apparent relationship.
```

Reply:
Oh you use numerical everywhere? That now looks/sounds weird to me.

Reply:
Do you prefer quantitative? I don't mind quantitative, but I don't like qualitative > categorical, and I don't like numeric + categorical or quantitative + categorical together, so I stuck with numerical + categorical pair. Do you have a specific preference for pair of terms?

Reply:
I can just live with numerical.